settings.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencyResolutionManagement {
 enableFeaturePreview("VERSION_CATALOGS")
 
 include(
+    ":appyx-interactions",
```

Comment:
Why is this outside of the libraries/samples folders?

Reply:
I imagine splitting to two top-level modules, appyx-interactions and appyx-navigation. Then you can have libraries/samples inside of each. I haven't gone the full way yet with appyx-navigation to unblock us working on the interactions.

Reply:
For now it will be used as a draft place to unblock us. We can always rearrange as we go. No harm is done, this isn't affecting 1.x or publication.